### DISCUSSION TITLE
Please develop this!!

### DISCUSSION THREAD

**Discussion by joereith** [ainfosec/FISSURE#6] [General]
* Distinguish between multiple hackrf devices (serial?)

I think this would be a nice feature to have seeing as the open source releasing of FISSURE is going to draw a lot of attention. Once people start to use it they will most likely be connecting all sorts of SDR's and giving FISSURE a spin. This will allow the easy visual separation of hackrf's by serial number to be able to assign individually to individual components. Having a list or dropdown box containing each detected hackrf in the GUI will ensure people are selecting the proper hackrf.

**Comment by cpoore1** [comment]
I think it does what you want now. See https://github.com/ainfosec/FISSURE/issues/7. Currently, there aren't many HackRF attacks. I'll work on filling those out.
